import type { UseCaseError } from "./use-case-error";

export class ResourceNotFoundError extends Error implements UseCaseError {
	constructor() {
		super("Recurso n√£o encontrado.");
	}
}
